src/App.tsx

```diff
@@ -44,6 +45,7 @@ export default function App() : JSX.Element {
     return <AppLoading />;
```

Comment:
Now that we use a splashscreen, is the <AppLoading /> part really necessary?

Reply:
No. We just need to make sure that we dont call SplashScreen.hide() before everything is loaded